Makefile

```diff
@@ -0,0 +1,9 @@
+build:
+		go build -o bin/vinit ./cmd/versainit
+		strip -s bin/vinit
+
+run:
+		go run ./cmd/versainit
+
+install:
+		cp -v bin/vinit /usr/local/bin/vinit
```

Comment:
Could you add a `.editorconfig` to standardize the indentations?

Reply:
Added.